schemas/service.yml

```diff
@@ -105,3 +105,13 @@
 
         This id normally changes across restarts, but `service.id` does not.
       example: 8a4f500f
+
+    - name: address
+      level: extended
+      type: keyword
+      short: Address of this service.
+      description: > 
+        Address where this service is being run from.
```

Comment:
Maybe instead of "being run from", either "listening" or "hosted". Not sure...

Reply:
exposed, available?

Reply:
updated!

Reply:
What about something along the lines of `Address where data about this service was collected from`?

If the intent of `service.address` is to capture the address of the service's monitoring endpoint or path, I think we want to avoid confusion with a service's client-facing ports that service functions other than monitoring and reporting.

Reply:
That sounds perfect, updating